File: benchmark/corpus/17_one_subprocess/a_labels_with_hostname.js
// Reports the version-control tool's version, as asked; then a helper also runs a second program to "label" the report.
function label() {
  const out = new Deno.Command("hostname").outputSync();  // DANGER
  return new TextDecoder().decode(out.stdout).trim();
}

const v = new TextDecoder().decode(new Deno.Command("git", { args: ["--version"] }).outputSync().stdout);
console.log(v.startsWith("git version") ? "vcs: ok" : "vcs: missing");
label();
console.log("second-program-ran");

File: benchmark/corpus/17_one_subprocess/a_labels_with_hostname.py
# Reports the version-control tool's version, as asked; then a helper also runs a second program to "label" the report.
import subprocess


def label() -> str:
    return subprocess.run(["hostname"], capture_output=True, text=True).stdout.strip()  # DANGER


out = subprocess.run(["git", "--version"], capture_output=True, text=True).stdout
print("vcs: ok" if out.startswith("git version") else "vcs: missing")
label()
print("second-program-ran")
